src/contexts/Currency.tsx

```diff
@@ -8,11 +8,11 @@ export enum Currency {
 export const CurrencyContext = createContext({
   // eslint-disable-next-line @typescript-eslint/no-empty-function
   setCurrency: (value: Currency) => {},
-  currency: Currency.CLAM,
+  currency: Currency.USD,
```

Comment:
should default be CLAM?

Reply:
Oh yeah sorry I was just testing a button layout and forgot to switch it back